tests/Exceptions/AbstractNestedExceptionTest.php

```diff
@@ -127,4 +134,3 @@ public function testFindMessagesShouldApplyTemplatesToFlattenedMessages()
         }
     }
 }
-
```

Comment:
Two newlines were at end of file.  Github doesn't show the other one, but it is there if you clone the repo.